### ISSUE TITLE
setup mechanism to push hotfixes to deployment

### ISSUE THREAD

**Issue by jaykay12** [jaykay12/Auth-API#24]
Create a workflow and document the way to provide hot-fixes to the deployment in the issue itself.

**Comment by jaykay12** [comment]
`master` - `production`+ some-under-development-work + hot-fix-work

```console
$ git checkout production
Switched to branch 'production'

$ git pull origin production
From https://github.com/jaykay12/Auth-API
 * branch            production -> FETCH_HEAD
Updating bd7f207..d56b5f6
Fast-forward
 helper-guide.md | 15 +++++++++++++--
 1 file changed, 13 insertions(+), 2 deletions(-)

$ git checkout -b hotfix
Switched to a new branch 'hotfix'

$ git cherry-pick -x 6650508
[hotfix f57a067] Unified bash file for running api for stage and dev
 Date: Sat May 9 18:13:53 2020 +0530
 3 files changed, 10 insertions(+), 9 deletions(-)
 delete mode 100644 service/start-dev.sh
 delete mode 100644 service/start-stage.sh
 create mode 100644 service/start.sh

$ git push origin hotfix
Enumerating objects: 6, done.
Counting objects: 100% (6/6), done.
Delta compression using up to 4 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 556 bytes | 556.00 KiB/s, done.
Total 4 (delta 2), reused 0 (delta 0)
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
remote: 
remote: Create a pull request for 'hotfix' on GitHub by visiting:
remote:      https://github.com/jaykay12/Auth-API/pull/new/hotfix
remote: 
To https://github.com/jaykay12/Auth-API.git
 * [new branch]      hotfix -> hotfix
```

---

Finally open a PR from hotfix  to production.

**Comment by jaykay12** [comment]
Closed by #25
